browser-test/src/delete_database.ts

```diff
@@ -1,7 +1,7 @@
-import { startSession, dropTables, endSession } from './support'
-
-module.exports = async () => {
-    const { browser, page } = await startSession();
-    await dropTables(page);
-    await endSession(browser);
-}
+import { startSession, dropTables, endSession } from './support'
+
+module.exports = async () => {
+  const { browser, page } = await startSession();
+  await dropTables(page);
+  await endSession(browser);
+}

```

Comment:
Suggestion (blocking): Since we're doing this we probably don't need the truncate tables script anymore: https://github.com/seattle-uat/civiform/blob/main/browser-test/bin/truncate_tables.sh

Reply:
Done.